keps/sig-apps/592-ttl-after-finish/README.md

```diff
@@ -237,7 +237,7 @@ Mitigations:
 
 ### Beta -> GA
 
-- Make a decision on wehther or not the feature should be extended to pods
+- ~~Make a decision on whether or not the feature should be extended to pods~~
```

Comment:
I'll handle this in a separate KEP/ alpha level implementation PR. You can for now say, TTL controller can be GA'ed without handling pods. The ability to extend TTL controller to work with pods can be introduced via a featuregate so that we can collect feedback and improve.

Reply:
Correct.

Reply:
addressed in my latest commit. Thanks @ravisantoshgudimetla